README.md

```diff
@@ -757,6 +763,50 @@ import foo from "bar";
 ```
 ___
 
+#### Changing a relative path to different directory
+In this example there is a relative path that should be changed to a sub-directory. This can be achieved like this (note: to use a function in `rename`, `select` must be `module`):
+
+###### Source Code
+```js
+import foo from "./path/to/bar.js";
+```
+
+###### Rollup Config
+```js
+plugins: [
+    importManager({
+        units: {
+            file: "**/my-file.js",
+            /** Example of using RegExp for module to match the exact module name. This is just an example use of RegExp and is not required for using a function in `rename` */
+            module: /^bar.js$/,
+            actions: {
+                select: "module",
+                /** moduleSourceRaw will be `"./path/to/bar.js"` including the double quotes */
+                rename(moduleSourceRaw) {

```

Comment:
This doesn't look right. You are calling the function 'rename' which isn't declared anywhere, rather than using the rename keyword and feeding it with a function.

Reply:
Actually this is a shorthand called a [Method definition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) :) you are welcome to change it if it feels too unclear.